docs/setup/navqplus.md

```diff
@@ -0,0 +1,180 @@
+# Connect Create® 3 to NavQ+ and set up ROS 2 Galactic
+
```

Comment:
Would you mind adding

!!! important
    **This is community-submitted content. Please feel welcome to submit PRs for additions or corrections.**
    
 here, please? Just to mirror NVIDIA's added setup doc.

Reply:
Great, just added it.